lib/heapwatch.js

```diff
@@ -47,16 +47,26 @@ class HeapWatch {
             this._gcStats = require('gc-stats')();
             this._gcStats.on('stats', this.reportStatsHandler);
             this.gcReportInterval = setInterval(() => {
-                const timings = {};
+                const gcMetrics = this.metrics.makeMetric({
+                    type: 'Gauge',
+                    name: 'gc',
+                    prometheus: {
+                        name: 'service_runner_heapwatch_gc_ns',
+                        help: 'heapwatch gc pause ns',
+                        staticLabels: { service: this.metrics.getServiceName() }
```

Comment:
Should this also be added automatically for other metrics created via service runner?  E.g. if I do app.metrics.makeMetric and use promtheus, perhaps this staticLabel; should somehow be auto added?  Maybe a app.makeMetric proxy?

Reply:
Based only on the [k8s citoid config](https://github.com/wikimedia/operations-deployment-charts/blob/master/charts/citoid/config/prometheus-statsd.conf), it appears this particular static label is being set for heapwatch and request duration metrics.  For Hyperswitch users, we intend to add this staticLabel in its request handler, but for projects based on service-template-node, they will likely have to set this attribute in their request handler.